src/pretalx/submission/migrations/0058_remove_question_required.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.1.8 on 2021-06-16 13:30
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("submission", "0057_auto_20210508_1308"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
If we do this, we would lose all data on existing questions in pretalx – we can't do this, it would break events that are running right now. There needs to be a data migration in between the two migrations that transfers the `required` state to the new `question_required` field.

If you need help with this, please let me know, and/or just drop the migration files from this PR, and I'll take care of it.

Reply:
We are not sure onhow to make the migration so we will just delete the files and we won't commit the last ones with the ```freeze_after``` field.